tests/testthat/test-ml-feature-one-hot-encoder.R

```diff
@@ -1,33 +1,32 @@
 context("ml feature one hot encoder")
 
 test_that("ft_one_hot_encoder() default params", {
-  skip_on_spark_master()
-  test_requires_latest_spark()
+  test_requires_version("3.0.0")
   sc <- testthat_spark_connection()
   test_default_args(sc, ft_one_hot_encoder)
 })
 
 test_that("ft_one_hot_encoder() param setting", {
-  skip_on_spark_master()
-  test_requires_latest_spark()
+  test_requires_version("3.0.0")
   sc <- testthat_spark_connection()
   test_args <- list(
     input_col = "foo",
     output_col = "bar",
+    input_cols = c("foo", "foo1"),
+    output_cols = c("bar", "bar1"),
     drop_last = FALSE
   )
   test_param_setting(sc, ft_one_hot_encoder, test_args)
 })
 
 test_that("ft_one_hot_encoder() works", {
-  skip_on_spark_master()
   sc <- testthat_spark_connection()
   iris_tbl <- testthat_tbl("iris")
   expect_equal(
     iris_tbl %>%
-      ft_string_indexer("Species", "indexed") %>%
+      ft_string_indexer("Species", "indexed", string_order_type = "alphabetDesc") %>%
```

Comment:
Modify this because of the behavior changes for StringIndexer. 
Check: https://github.com/apache/spark/blob/7d4eb38bbcc887fb61ba7344df3f77a046ad77f8/docs/ml-migration-guide.md#changes-of-behavior